dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/BufferManagerImpl.java

```diff
@@ -19,55 +19,81 @@
 
 import com.ibm.websphere.ras.Tr;
 import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.ffdc.annotation.FFDCIgnore;
 import com.ibm.wsspi.collector.manager.BufferManager;
-import com.ibm.wsspi.collector.manager.Handler;
 import com.ibm.wsspi.collector.manager.SynchronousHandler;
 
 public class BufferManagerImpl extends BufferManager {
 
-	private static final TraceComponent tc = Tr.register(BufferManagerImpl.class);
+	private static final TraceComponent tc = Tr.register(BufferManagerImpl.class);	
+    private Buffer<Object> ringBuffer;
 	private static final ReentrantReadWriteLock RERWLOCK = new ReentrantReadWriteLock(true);
 	private Set<SynchronousHandler> synchronizedHandlerSet = new HashSet<SynchronousHandler>();
```

Comment:
I know you didn't change this... but can you please rename synchronizedHandlerSet to synchronousHandlerSet?   They aren't synchronized! :-)